kingfisher_scrapy/base_spider.py

```diff
@@ -444,6 +444,9 @@ def next_link(self, response, **kwargs):
         if url:
             return self.build_request(url, formatter=self.next_page_formatter, **kwargs)
 
+        if self.from_date and self.until_date and (self.until_date - self.from_date).days <= 1:
```

Comment:
```suggestion
        if self.from_date and self.until_date:
```
the just one day was an example, I think we should check only if these parameters are set. @jpmckinney what do you think?

Reply:
The `MissingNextLinkError` exception is just to help developers identify when their spider is broken. I think we can return if any filters are set (not just dates).